src/lib389/lib389/cli_idm/account.py

```diff
@@ -235,6 +235,8 @@ def create_parser(subparsers):
 
     list_parser = subcommands.add_parser('list', help='list accounts that could login to the directory', formatter_class=CustomHelpFormatter)
     list_parser.set_defaults(func=list)
+    list_parser.add_argument('--full-dn', action='store_true',
```

Comment:
I'm not sure we need it here as it returns DN by default (and there's no way to return a name).
And I think it's okay as it returns all accounts (not only users). So just DN makes sense.

Reply:
> I'm not sure we need it here as it returns DN by default (and there's no way to return a name). And I think it's okay as it returns all accounts (not only users). So just DN makes sense.

I'm confused by your comment, it returns the RDN "value" by default _not_ the DN.  That's the whole point of this feature/PR is to have an option to list the DN's.